.github/workflows/vuln-check.yml

```diff
@@ -7,7 +7,8 @@ on:
     # The branches below must be a subset of the branches above
     branches: [ "master" ]
   schedule:
-    - cron: '10 2 * * 0'
+    - cron: '* 2 * * *'
```

Comment:
Isn't this every minute from `02:00` up to and including `02:59`?

Reply:
It is. I get it from cron config site generator... I should have double-check myself 😞